pkg/model/manifest.go

```diff
@@ -663,19 +663,21 @@ func Read(bytes []byte) (*Manifest, error) {
 
 	hasShownWarning := false
 	for _, dev := range manifest.Dev {
-		if (dev.Image.Context != "" || dev.Image.Dockerfile != "") && !hasShownWarning {
+		if dev.Image != nil && (dev.Image.Context != "" || dev.Image.Dockerfile != "") && !hasShownWarning {
```

Comment:
`dev.Image` was `nil` in case of 

```yaml
dev:
  frontend:
  backend:
```

so this extra check makes it safer and removes the panic generated from this line when accessing `dev.Image.Context`.